test/live/live_test.go

```diff
@@ -39,3 +39,187 @@ func TestLiveCoinPricesFromCoingecko(t *testing.T) {
 			requiredEnvVars).
 		Run(filepath.Join(scriptDir, projectName+".txtar"))
 }
+
+func TestLiveErrorHandlingAttestFnCall(t *testing.T) {
```

Comment:
Pretty straightforward. Almost exact copies of our existing tests except that these require env vars for connecting to a remote bky-as server